src/orders/components/OrderTransactionPayment/OrderTransactionPayment.tsx

```diff
@@ -70,7 +69,7 @@ const OrderTransactionPayment: React.FC<OrderTransactionPaymentProps> = ({
       if (action === TransactionActionEnum.CHARGE) {
         onCapture();
       }
-      if (action === TransactionActionEnum.VOID) {
+      if (action === TransactionActionEnum.CANCEL) {
         onVoid();
```

Comment:
Should we also rename this function to `onCancel`?

Reply:
Probably but I leave it because this requires bigger refactor